src/Event/EventType/Korbo/cs.yaml

```diff
@@ -194,7 +194,7 @@ email:
         buttonNotWorking: "Pokud přihlašovací tlačítko nefunguje nebo není vidět, zkopíruj a vlož tento odkaz do vyhledávacího řádku svého prohlížeče: "
     closed:
         subject: "uzamčeno"
-        successfullySent: "Úspěšně jsi poslal/a svou přihlášku na %event.getReadableName%. Nyní musíme zkontrolovat, zda je všechno v pořádku, a pokud je, pošleme ti platební údaje. Mohlo by to trvat i několik dní, buď tedy trpělivý/á, prosím. :)"
+        successfullySent: "Úspěšně jsi poslal/a svou přihlášku na KORBO 2022. Nyní musíme zkontrolovat, zda je všechno v pořádku, a pokud je, pošleme ti platební údaje. Mohlo by to trvat i několik dní, buď tedy trpělivý/á, prosím. :)"
```

Comment:
Je nutné rušit dynamičnost? Takhle to nepůjde použít pro další ročníky

Reply:
Čitelné v template, nebo pro účastníka? Zauvažuj ještě nad tou změnou. Jestli je cílem používat všude kapitálky, pojďme do toho všude nastavením v DB